starport/pkg/chaincmd/chaincmd.go

```diff
@@ -270,6 +275,24 @@ func (c ChainCmd) ShowNodeIDCommand() step.Option {
 	return step.Exec(c.appCmd, c.attachHome(command)...)
 }
 
+// SetConfigCommand returns the command to set config value
+func (c ChainCmd) SetConfigCommand(name string, value string) step.Option {
+	// Check version
+	if c.isStargate() {
+		return nil // not defined for Stargate
```

Comment:
I think it makes sense to `panic` in this func and also in `RestServerCommand` if `ChainCmd` is for Stargate.

I kind of prefer keeping `Launchpad` prefix as well for LP only funcs to prevent confusions.